frontend/views/containers/group-settings/GroupRulesSettings.vue

```diff
@@ -58,6 +71,12 @@ export default ({
       // because the group disappears. it's not a big deal though
       return this.groupProposalSettings() || {}
     },
+    votingRulesSorted () {
+      // NOTE: temporarily hiding DISAGREEMENT_RULE from the settings.
```

Comment:
@taoeffect just reverted the code back to what they were and hided the 'disagreement-rule' by making this change here with the comment explaining the context. Thought this is the way that requires the least amount of code change, hence making things easier when we want to re-enable it. but pls let me know if you have other thoughts.
cheers,